source/contentscript.js

```diff
@@ -1,11 +1,14 @@
-// when asked tell the extension about the SPDY/HTTP2 status of current page
-
+// determineConnectionInfo returns the protocol used to connect to the first hop
+// for the first navigation event available.
 function determineConnectionInfo() {
-  var loadTimes = chrome.loadTimes();
-  return {
-    spdy: loadTimes.wasFetchedViaSpdy,
-    type: loadTimes.npnNegotiatedProtocol || loadTimes.connectionInfo
-  };
+  if (performance && performance.getEntriesByType) {
+    let entry = performance.getEntriesByType('navigation')[0];
+    let proto = entry && entry.nextHopProtocol;
```

Comment:
Looking at the [MDN browser compatibility table](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/nextHopProtocol#Browser_compatibility) shows that only Firefox currently supports `nextHopProtocol`, so we should probably keep both methods for now